webapp/src/services/refunds.ts

```diff
@@ -0,0 +1,9 @@
+import { Transaction } from '../types';
+import axios from 'axios';
+
+export const Refunds = {
+  async initiateRefund () {
```

Comment:
I think it would make more sense for this method to go in `webapp/src/services/transactions`

Reply:
I did this to be consistent with `Donations`. One could make the same argument about donations, but in this case I didn't want to change too many things and decided to be consistent with what's already there.